### ISSUE TITLE
After docker crashed there's no way to restore challenges.

### ISSUE THREAD

**Issue by thefishermanhacker** [juice-shop/juice-shop#302]
Hi,

Its probably had nothing with docker but after my docker crashed because of bug  #301 and after start it again the challenges gone.
I verified the cookies and saw `continueCode BP4graVEq7DZowBYQPyNljOKex9djrHMdmv8rXWR562gpJ4b13knMzLv3ERx`.

Scenario:
1. Start jucie-shope via a docker.
2. Do 2 or more challenges.
3. Save your progress, (SPOILER :)).
4. Verify the cookie created.
5. Crash the docker via #301 
6. Start the docker as `docker run -d -p 3000:3000 bkimminich/juice-shop`
7. Refresh the page.
You'll get 2 scenarios:
1. The challenges don't get update.
2. Your challenges will be displayed.

Thanks.

**Comment by bkimminich** [comment]
I just played it through, and I'm pretty sure it is some issue not related to either Docker or the Juice Shop app.

1. Started `juice-shop:latest` via Kitematic
2. Solved four challenges and saved progress on Score Board
3. Killed container with `' union select name,2,3,4,null from sqlite_master--`
4. Closed browser window
5. Restarted container via Kitematic
6. Opened browser, visited Score Board
7. Clicked "Restore" button
8. **All previously solved challenges were restored**

Is your browser killing cookies when closed or anything? Might some extension/plugin interfere? I tested with Chrome.
